java-baseball/src/main/java/baseball/domain/Judgment.java

```diff
@@ -0,0 +1,29 @@
+package baseball.domain;
+
+public class Judgment {
+
+    private final NumbersComparator comparator = new NumbersComparator();
+
+    private int ball;
+    private int strike;
+
+    public void judge(Computer computer, Player player) {
+        ball = comparator.compareSameNumber(computer, player);
+        strike = comparator.compareSamePlace(computer, player);
```

Comment:
```suggestion
        ball = computer.compareSameNumber(player);
        strike = computer.compareSamePlace(player);
```
바로 객체로 넘겨줘도 될 것 같다는 생각이듭니다!! 쬴리님 생각은 어떠신가요?